mteb/models/repllama_models.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import annotations
+
+import logging
+from typing import Any, Callable, Literal
+
+import numpy as np
+import torch
+import torch.nn.functional as F
+import tqdm
+from transformers import AutoModel, AutoTokenizer
+
+from mteb.encoder_interface import Encoder
+from mteb.model_meta import ModelMeta
+from mteb.models.text_formatting_utils import corpus_to_texts
+
+logging.basicConfig(level=logging.WARNING)
+logger = logging.getLogger(__name__)
+
+EncodeTypes = Literal["query", "passage"]
+
+
+class RepLLaMAWrapper:
+    def __init__(self, *args, **kwargs):
+        try:
+            from peft import PeftModel
+        except ImportError:
+            raise ImportError(
+                "To use the RepLLaMA based models `peft` is required. Please install it with `pip install peft`."
+            )
```

Comment:
```suggestion
                "To use the RepLLaMA based models `peft` is required. Please install it with `pip install 'mteb[peft]'`."
            )
```

that way we can specify the version